### ISSUE TITLE
[qwik-pod] Showcase Clean Up

### ISSUE THREAD

**Issue by kenslachtajr** [thisdot/starter.dev-github-showcases#1034]
We need a general proofing and cleanup of the work that has been done to make sure there are no errors or omissions. 

## Acceptance Criteria

- [ ] Review the current showcase is free of errors, and everything expected has been included and is operating as expected.

Timebox: max 2 hours each

## Who has completed this:
- [x] Daian
- [x] Chinedu
- [x] Mattia
- [x] Mark

**Comment by markshenouda** [comment]
## I have a few notes on the Qwik starter kit:

- [ ] This Tailwind pattern is against the official recommendations from Tailwind team, it's preferred to use the tailwind classes directly inside the JSX
![Image](https://user-images.githubusercontent.com/49324987/206570693-c9bdff71-1cbc-4c0b-924f-64e1b9237cc5.png)

- [ ] Also adding the prettier tailwind plugin, really improves the DX in tailwind projects

- [ ] There is also an error in the data fetching page
![Image](https://user-images.githubusercontent.com/49324987/206571303-d25dedfc-d878-4df2-9c9b-824909198f5d.png)


- [ ] I see there is `paths` array without `baseUrl`, we should add it and use the absolute passes (preferred) or delete `paths`

![Image](https://user-images.githubusercontent.com/49324987/206571539-c519de13-b548-42fb-a1f8-aaa4f5bb05de.png)

- [ ] We don't need to create new components to export existing ones

![Image](https://user-images.githubusercontent.com/49324987/206571901-86917d64-14cf-4c8b-a054-b2e26d051ef2.png)

we can use the trick we used in other starters by adding `export * from './MyComponent'` in the `index.tsx` instead, much cleaner and matches the other kits

- [ ] Also add the `index.ts` from the previous point for the components so we don't have to type them twice each time we import them

![Image](https://user-images.githubusercontent.com/49324987/206572865-a1f104f9-fc4f-46f6-a61f-1a73a11d9954.png)

**Comment by SDaian** [comment]
- After navigate trough Pagination in the Profile View, the Pagination component disappear:

Before clicking Prev/Next
![image](https://user-images.githubusercontent.com/14813235/211540827-b4d629df-8213-4ac0-87ad-e4980b2bbd29.png)

After clicking Prev/Next
<img width="1085" alt="image" src="https://user-images.githubusercontent.com/14813235/211540944-c0dffe83-473d-403c-b546-babd51f06d42.png">

**Comment by Megio** [comment]
When we set a label filter issues and pr are filtered in the right way.
But when we try to set up another label filter, we don't have all the options except for the options we selected before.
This is the same behavior as the next.js showcase, but I don't know if it is the behavior that we want.

![Screenshot 2023-01-10 at 12 37 48](https://user-images.githubusercontent.com/32470461/211542825-bc6ab060-8a22-473b-8188-9b938c93e63b.png)

![Screenshot 2023-01-10 at 12 37 57](https://user-images.githubusercontent.com/32470461/211542849-4a197669-c7d3-4138-80df-e1613325b55c.png)
